test files/ModuleTests/SerialTestConnection/SerialTestConnectionTests.sbs

```diff
@@ -0,0 +1,74 @@
+using StepBro.Streams;
```

Comment:
Would be nice to have a test, that checks that the same hash is generated twice in a row for the same code.